data/map.txt

```diff
@@ -21145,6 +21145,7 @@ system "Sagittarius A*"
 	government Uninhabited
 	habitable 10000
 	haze _menu/haze-full
+	arrival 500 jump
```

Comment:
To me, a more natural syntax would be `[<descriptor>] <#value>`, or a child node sequence e.g.

```sh
arrival [<#default value for all types>]
  <type> <#value for just this type> # <type> may be e.g. "link", "jump", "planet"
  ...
```

Changing to this descriptor-first format can likely simplify the parsing code, too.

Reply:
The keywords/syntax used in PR are based on the descriptions in this [RFC](https://github.com/EndlessSkyCommunity/endless-sky/pull/51). (Sorry that I forgot to include the link.)

Should we discuss the syntax in the RFC PR instead of here?

Reply:
Updated to the child node sequence syntax.